integration-test/tests/email-autofill.macos.spec.js

```diff
@@ -279,30 +279,27 @@ test.describe('macos', () => {
     })
     test.describe('matching performance', () => {
         test('real-world form', async ({page}) => {
-            await forwardConsoleMessages(page)
```

Comment:
I've removed the console forwarding from these tests so we can more easily read the performance results of the test itself without digging through lots of noise.